### ISSUE TITLE
fit$print()

### ISSUE THREAD

**Issue by nipnipj** [stan-dev/cmdstanr#741]
Guys, please confirn that `fit$print()` is working. I get the following output
```
variable                 mean               median sd mad
 lp__       -\033[31m935\033[39m -\033[31m935\033[39m  2   2
 alpha        -\033[31m1\033[39m   -\033[31m1\033[39m  0   0
 alpha2 
```

while `fit$summary()` works fine.

```
variable  mean median     sd    mad    q5   q95  rhat ess_bulk ess_tail
  <chr>    <num>  <num>  <num>  <num> <num> <num> <num>    <num>    <num>
1 yfc[1]    1.68   1.68 0.0662 0.0687  1.57  1.78  1.00     543.     642.
2 yfc[2]    1.35   1.35 0.108  0.108   1.19  1.53  1.01     402.     719.
```

**Comment by rok-cesnovar** [comment]
This is related to https://github.com/stan-dev/posterior/issues/275 I think.

**Comment by andrjohns** [comment]
Verified that the issue was resolved with the release of `posterior 1.4.1`
